Reply/app/src/main/res/color/star_statelist.xml

```diff
@@ -12,12 +12,11 @@
   or implied. See the License for the specific language governing permissions and limitations under
   the License.
   -->
-<resources>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
nit: Can you follow the formatting so this file matches the existing color resource files? It should look like this:
https://github.com/material-components/material-components-android-examples/blob/develop/Reply/app/src/main/res/color/color_navigation_drawer_menu_item.xml